modular_citadel/code/modules/festive/vectorvariants.dm

```diff
@@ -0,0 +1,45 @@
+/obj/vehicle/sealed/vectorcraft/cyber
+	name = "Cyberscuttle"
```

Comment:
```suggestion
	name = "Cybershuttle"
```
Unless the mispelling was intentional...

Reply:
It scuttles! I couldn't think of another name for it's inspirer